Anchorage.swift

```diff
@@ -488,50 +497,60 @@ public struct EdgeConstraints {
             }
         }
     }
-
+    
 }
 
 // MARK: - Constraint Builders
 
 private struct ConstraintBuilder {
-
+    
     typealias Horizontal = (NSLayoutXAxisAnchor, LayoutExpression<NSLayoutXAxisAnchor>) -> NSLayoutConstraint
     typealias Vertical = (NSLayoutYAxisAnchor, LayoutExpression<NSLayoutYAxisAnchor>) -> NSLayoutConstraint
-
+    
     var top: Vertical
     var leading: Horizontal
     var bottom: Vertical
     var trailing: Horizontal
-
+    
+    #if swift(>=3.0)
     init(horizontal: @escaping Horizontal, vertical: @escaping Vertical) {
```

Comment:
I think you can move the `@escaping` and `#if` into the typealias. Might clean this up a little